_includes/releases/cloud/2023-05-15.md

```diff
@@ -0,0 +1,16 @@
+## May 15, 2023
+
+In addition to many of the Feature Highlights in the [CockroachDB v23.1.0 Release Notes](v23.1.html), the following features are specifically available to Cloud clusters once they are upgraded:
+
+<h3> General changes </h3>
+
+- {{ site.data.products.serverless }} multi-region is now publicly available in [preview](../{{site.versions["stable"]}}/cockroachdb-feature-availability.html#feature-availability-phases). Multi-region benefits like reduced latency and high availability are now achievable in conjunction with the ease-of-operation and automatic scaling advantages of the {{ site.data.products.serverless }} deployment model.
```

Comment:
Suggest making first use of "multi-region" a link to https://www.cockroachlabs.com/docs/v23.1/multiregion-overview.html